src/TMap.cpp

```diff
@@ -45,8 +45,8 @@
 
 
 TMap::TMap(Host* pH, const QString& profileName)
-: mDefaultAreaName(tr("Default Area"))
-, mUnnamedAreaName(tr("Unnamed Area"))
+: mDefaultAreaName(tr("Default area"))
```

Comment:
Can you confirm that you're replaced all instances of this text? I have a feeling some code might depend on this special area name. Same for the one below.

Reply:
Can't positively confirm as these may even be in use in player scripts in Lua.. See prior discussion. https://github.com/Mudlet/Mudlet/pull/5523#discussion_r730327860

Maybe just revert then and leave capitalized?

Reply:
Player scripts in Lua would be fine I think, just looking out for Mudlets own use of these.

Reply:
TBH That is why I defined them - so that the class variable (or is it a constant, I didn't check) could be used consistently through the (C++) code...